staging/src/k8s.io/component-helpers/node/util/cidr.go

```diff
@@ -50,7 +50,7 @@ func PatchNodeCIDRs(c clientset.Interface, node types.NodeName, cidrs []string)
 	if err != nil {
 		return fmt.Errorf("failed to json.Marshal CIDR: %v", err)
 	}
-	klog.V(4).Infof("cidrs patch bytes are:%s", string(patchBytes))
+	klog.FromContext(ctx).V(4).Info("cidrs patch bytes", "patchBytes", string(patchBytes))
```

Comment:
Now that `PatchNodeCIDRs` accepts a context, you should also use it below instead of `context.TODO`.

I was about to say "just pass a klog.Logger", but because the function needs it context, it's the better choice.